iceoryx_hoofs/include/iceoryx_hoofs/internal/cxx/convert.inl

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2019, 2021 by Robert Bosch GmbH. All rights reserved.
-// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+// Copyright (c) 2021 - 2022 by Apex.AI Inc. All rights reserved.
```

Comment:
I would change the log level in this file to `INFO`. I think it's not a severe error when the conversion fails and that this can happen is also indicated by the return type.

Reply:
@FerdinandSpitzschnueffler `INFO` seems too low. Something has in fact gone wrong. What about `WARN` ?

Reply:
cc @elBoberido

Reply:
@ithier I see your point but in this case it is actually a debug message. The developer is already informed by the return value that the conversion failed. Having a log entry is not even required on this low level function but also does not harm. On the other hand, having a log message with `Error` or `Warn` although the developer already took care of the fault is misleading and also scary. I tend more and more to just make this a debug log message.